README.md

```diff
@@ -13,19 +13,59 @@ sudo apt-get install python3 foobar
 
 # System Setup #
 
-## Passwordless SSH and Sudo Access #
+## Passwordless SSH and Sudo Access ##
+
+Note: After key distribution, the machine in question has to be restarted (or
+a logout has to happen).
 
 ```
-echo fff > fff
+alpha, beta, koppa:
+sudo visudo
+
+Append to the end of file of alpha, beta and koppa (careful, this is a
+possible security leak!):
+USER ALL=NOPASSWD: ALL
+
+Use RSA key pair for remote login:
+alpha:
+ssh-keygen
+ssh-copy-id user@beta
+ssh-copy-id user@koppa
+
+Remote log in once for key unlocking (to both servers), e.g. (perform LOGOUT
+or RESTART first!):
+ssh ["beta", "koppa"]@localhost sudo sysctl -a > /dev/null
 ```
 
 ## Static Route Configuration and Static ARP Entries #
 
 Configure static routes and add static ARP entries. Static
-arp entries are required to reduce measurement noise.
+arp entries are required to reduce measurement noise. Also, enable ip forward
+on the router.
+Note: These changes will be lost on a restart.
 
 ```
-ip route add ...
+alpha:
+[sudo] ifconfig enp4s0 down
+[sudo] ip a add 10.0.0.1/16 dev enp4s0
+[sudo] ip r add default via 10.0.0.205
+[sudo] ifconfig enp4s0 up
+
+beta:
+[sudo] ifconfig eth0 down
+[sudo] ip a add 10.0.1.1/16 dev eth0
+[sudo] ip r add default via 10.0.1.205
+[sudo] ifconfig eth0 up
```

Comment:
ip and ifconfig mixed? ```ip link set dev enp4s0 up```?

Where are the static ARP entries documented to reduce arp queries?